docs/snippets/jmespath/powertoolsBase64GzipJmespathPayload.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think this would benefit from having the ‘raw’ form as a comment either in this file or in the sample

Reply:
I have added it in the sample using the annotation feature, which looks like this:

<img width="863" alt="Screenshot 2024-03-24 at 20 26 45" src="https://github.com/aws-powertools/powertools-lambda-typescript/assets/7353869/5be5c3f2-459a-458c-a4cf-6a07229aaad1">

Reply:
Did the same [as here](https://github.com/aws-powertools/powertools-lambda-typescript/pull/2187#discussion_r1536891713).